README.md

```diff
@@ -848,6 +848,25 @@ const Screen = () => (
 )
 ```
 
+### 7.5 Styled Component Naming Convention
```

Comment:
Maybe we also could add a text to always use `import * as S from` to import SC? 😛

Reply:
@felipefialho perfect, I've included it: f710432d6a7ff2fc3c61a05a5d1960e1791278c4